src/app/services/authentication.service.ts

```diff
@@ -60,56 +60,24 @@ const auth0WebConfig : IonicAuthOptions = {
   providedIn: 'root'
 })
 export class AuthenticationService extends IonicAuth {
```

Comment:
The passing of the loading indicator to the service was weird at best, but also not needed as under normal circumstances the handleCallback() and login() will either resolve or reject.

In the one case where login() does not resolve or reject, there is also no event here, so 🤷🏻‍♂️